docs/network/default_cni_plugin.md

```diff
@@ -149,3 +150,44 @@ The specified interface will be added in OVS bridge `br-ex` which acts as gatewa
 microshift-ovs-init.service is able to setup one additional host interface for cluster ingress/egress traffic.
 This is done by specifying the `externalGatewayInterface` in the CNI config file `/etc/microshift/ovn.yaml`.
 The external gateway interface will be added in a second OVS bridge `br-ex1`. Cluster pod traffic destinated to additional host subnet will be routed through `br-ex1`.
+
+### NodePort host addresses
```

Comment:
Could we reword this?
I feel that something like "Blocking access to NodePort from/for specific host interfaces" would better describe intent of the section and the instructions here. What do you think?

Reply:
Done.